packages/components/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 -   `SlotFill`: rewrite the non-portal version to use `observableMap` ([#67400](https://github.com/WordPress/gutenberg/pull/67400)).
 -   `DatePicker`: Prepare day buttons for 40px default size ([#68156](https://github.com/WordPress/gutenberg/pull/68156)).
 -   `SlotFill`: register slots in a layout effect ([#68176](https://github.com/WordPress/gutenberg/pull/68176)).
+-   `DropZone`: Make the drop zone in Storybook the same size as the item ([#68231](https://github.com/WordPress/gutenberg/pull/68231)).
```

Comment:
I think a CHANGELOG entry is not needed for a minor story change, but happy to leave it to your judgment.

Reply:
I see, I'll delete it and then merge it 👍